### PR TITLE
Add node source examples

### PR DESCRIPTION
**Pull request by jrochas**
- add node source json examples in the catalog, under the kind NodeSource

### CODE REVIEW

File: NodeSources/METADATA.json [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+					"kind": "nodesource",
+					"commitMessage": "First commit",
+					"contentType": "application/json"
+				},
+				"file" : "resources/catalog/OpenStack.json"
+			},
+			{
+				"name" : "VMware",
+				"metadata" : {
+					"kind": "nodesource",
```
Comment (mbenguig) [inline NodeSources/METADATA.json]: (nodesource or NodeSource)

Comment (jrochas) [reply NodeSources/METADATA.json]: I have confirmation from Chloé and that the kind is lowercased in any case.

Comment (ShatalovYaroslav) [reply NodeSources/METADATA.json]: @jrochas I give the confirmation that in catalog service and catalog portal the kind can be searched case-insensitive. But to be in line with naming for all other kinds that we have in official list from our managers, please put "NodeSource".

Comment (jrochas) [reply NodeSources/METADATA.json]: @ShatalovYaroslav ok, I see your point, no problem I will change it.

**Pull request merged**